File: python/1045.py
import sys
e = list(map(float, input().split(' ')))

e = sorted(e,reverse=True)

a = e[0]
b = e[1]
c = e[2]


if a >= b+c:
    print('NAO FORMA TRIANGULO')
    sys.exit()
if a**2 == b**2+c**2:
    print('TRIANGULO RETANGULO')
if a**2 > b**2+c**2:
    print('TRIANGULO OBTUSANGULO')
if a**2 < b**2+c**2:
    print('TRIANGULO ACUTANGULO')
if a == b and b == c:
    print('TRIANGULO EQUILATERO')
elif a == b or a == c or b == c:
    print('TRIANGULO ISOSCELES')

File: php/PROBLEMS/1045.php
<?php

$e = fgets(STDIN);
$e = explode(' ', trim($e));
$e = array_map('floatval', $e);
rsort($e);
$a = $e[0];
$b = $e[1];
$c = $e[2];


if ($a >= $b + $c)
{
    printf("NAO FORMA TRIANGULO\n");
    return 0;
}
if ($a * $a == ($b * $b + $c * $c))
{
    printf("TRIANGULO RETANGULO\n");
}
if ($a * $a > ($b * $b + $c * $c))
{
    printf("TRIANGULO OBTUSANGULO\n");
}
if ($a * $a < ($b * $b + $c * $c))
{
    printf("TRIANGULO ACUTANGULO\n");
}
if ($a == $b && $b == $c)
{
    printf("TRIANGULO EQUILATERO\n");
}
else if ($a == $b || $a == $c || $b == $c)
{
    printf("TRIANGULO ISOSCELES\n");
}

?>
